src/stepper/Stepper.tsx

```diff
@@ -187,7 +195,10 @@ export default class Stepper extends PureComponent<StepperProps, StepperState> {
     return (
       <div
         className={cc(['kirk-stepper', `kirk-stepper-${display}`, className])}
+        ref={this.containerRef}
         aria-label={title}
+        // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
+        tabIndex={0}
```

Comment:
I think if you use `tabIndex={-1}` you don't need the ESlint disable, it would still be able to get the focus programmatically but not by user navigation.